### PR TITLE
lots of mobile improvements

### LINKED ISSUE #7183: more padding bottom on standalone mobile PWA
use 

`@media (display-mode: standalone)`

### PR DESCRIPTION
**Pull request by PeerRich**
- added pwa: extension for standalone 
- lots of padding improvements on mobile

### CODE REVIEW

File: packages/config/tailwind-preset.js [hunk 1]
```diff
@@ -140,6 +140,9 @@ module.exports = {
           900: "#f3f4f6",
         },
       },
+      screens: {
+        pwa: { raw: "(display-mode: standalone)" },
+      },
```
Comment (sean-brydon) [inline packages/config/tailwind-preset.js:145]: I didnt even know this was possible :D

**Review by sean-brydon — APPROVED** [review]
Tested on a few devices on sizzy looks good :)

File: packages/config/tailwind-preset.js (hunk 1 above)
Comment (PeerRich) [reply packages/config/tailwind-preset.js:145]: saw this in another project and had to steal it

Comment (PeerRich) [reply packages/config/tailwind-preset.js:145]: the less global.css the better

**Pull request merged**